task 06/task6-1/ConsoleApp4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace ConsoleApp4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            
+            var n = Program.delenie("n");
+            
+            Console.WriteLine(n);
+        }
+
+        public static bool delenie (string x)
```

Comment:
Традиционно методы называют с большой буквы
```suggestion
        public static bool Delenie (string x)
```